boa_engine/src/vm/mod.rs

```diff
@@ -40,6 +44,7 @@ pub struct Vm {
     pub(crate) frames: Vec<CallFrame>,
     pub(crate) stack: Vec<JsValue>,
     pub(crate) err: Option<JsError>,
+    #[cfg(feature = "trace")]
     pub(crate) trace: bool,
```

Comment:
Do we still need the trace field when we can use `#[cfg(feature = "trace")]` and `#[cfg(not((feature = "trace"))]`?

Reply:
Hmm I guess this is to enable disabling tracing for particular CLI requests even if it's enabled for other requests in the same executable.

Reply:
The `trace` feature enables the ability to trace (compile time), and the `trace` field turns the trace on/off at runtime :)

clippy would also complain that there is an unused field.